src/configurations/destinations/amazon_audience/ui-config.json

```diff
@@ -0,0 +1,150 @@
+{
+  "description": "GENERATED CODE DO NOT EDIT",
```

Comment:
why this line?
```suggestion
  "description": "GENERATED CODE DO NOT EDIT",
```

Reply:
Just using the template it doesn't have any usage. Shall I remove it ?

Reply:
Please remove this

Reply:
@anantjain45823 this is not removed yet

Reply:
removed now